src/RefreshSpinner/RefreshSpinner.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { ActivityIndicator } from 'react-native';
+import styles from './RefreshSpinner.styles';
+
+const RefreshSpinner = ({ CustomComponent }) => {
+  if (CustomComponent) {
+    return <CustomComponent style={styles.spinner} />;
+  }
+
+  return <ActivityIndicator size="large" color="blue" style={styles.spinner} />;
```

Comment:
the color here is very specific, should we avoid rendering the default spinner?

Reply:
Only allow the custom component? --> would have to be required when `isRefreshing={true}`

Reply:
To be honest my solution looks ugly now that I see it :laughing:, more ideas are welcomed

Reply:
I simplified this, see comment below